CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contribution Guidelines
+
+Thank you for your interest in contributing to dwave-examples! Before getting
+started, we encourage you to take a look at this document of guidelines.
+
+## What We're Looking For
+
+While we have a preference for application oriented examples, we welcome all
+examples that demonstrate use of Ocean&#8482; SDK tools.
+
+To see our categorized collection of demos, create a free account on the
+[Leap&#8482; Quantum Cloud Service](https://cloud.dwavesys.com/leap/signup/) and take a look at
+[our library of examples](https://cloud.dwavesys.com/leap/examples/) or
+go through the demo code repositories in [dwave-examples](https://github.com/dwave-examples).
```

Comment:
You can see the **categorized** collection in https://cloud.dwavesys.com/leap/examples, but https://github.com/dwave-examples is uncategorized (at least in any user-friendly way)

Additionally, it's not clear how this paragraph adds to describing "What We're Looking For". Do you mean to say that they should learn what other examples look like or avoid duplications? Can we add a sentence that makes their survey useful for contributing?

Reply:
Thanks for your feedback! I've make the remaining changes in this commit https://github.com/dwave-examples/template-dash/pull/1/commits/b2f4bc7ada3821673e413099c995af2f91560a27 Let me know what you think!